src/main/java/com/ldtteam/structurize/util/ChangeStorage.java

```diff
@@ -46,7 +47,7 @@ public class ChangeStorage
     /**
      * The operation which was done
      */
-    private final String operation;
```

Comment:
Uff, do we really want logic based on components? or is this purely UI (could be better variable name then)?

Reply:
Purely used for UI at this point.